input/pages/introduction.md

```diff
@@ -76,7 +76,7 @@ This Implementation Guide (Figure 2-1(b)) is the successor of the CQL-based HQMF
 #### CQL-based HQMF IG R1 STU4.1
 {: #cql-based-hqmf-ig-r1-stu4.1}
 
-The first version of the CQL-based HQMF IG was released in September 2015 and was intended to be used in conjunction with the pre-existing QDM based HQMF R1 IG. Since 2015, the community and the standards have evolved and the current version of QDM (v5.4) no longer contains expression logic, ceding this functionality to CQL. The CQL-based HQMF IG is the sole guide describing how to use QDM, CQL, and HQMF in combination (Figure 2-1a).
+The first version of the CQL-based HQMF IG was released in September 2015 and was intended to be used in conjunction with the pre-existing QDM based HQMF R1 IG. Since 2015, the community and the standards have evolved and beginning from the current version of QDM (v5.6) no longer contains expression logic, ceding this functionality to CQL. The CQL-based HQMF IG is the sole guide describing how to use QDM, CQL, and HQMF in combination (Figure 2-1a).
```

Comment:
Change from "beginning from the current version of QDM (v5.6) no longer contains expression logic ..."
to 
"beginning from the version 5.4, QDM no longer contains expression logics, ..."